### DISCUSSION TITLE
Купил линцезию, а она пишет не подходит.

### DISCUSSION THREAD

**Discussion by DimonPro242** [aclap-dev/video-downloadhelper#1463] [License & Purchase requests]
### What's your issue?

Здравствуйте, Пользуюсь Яндекс Браузер и в нем установил расширения и через расширение перешел на сайт и купил лицензию, ну при активации пишет <Лицензия для Firefox, но сборка браузера для Chrome> оказался что я не то что нужно купил? Можете помочь 

### The 10 first digits (only 10) of your license number or email address

a7609fb52f

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
We only support Chrome, Firefox and Edge.
If you need a refund, tell me.
If you need to purchase for another browser,  all the links are here: https://github.com/aclap-dev/video-downloadhelper/wiki#go-premium

**Reply by DimonPro242** [reply]
Можно тогда пожалуйста Сделать возврат средств?
Я сейчас тогда куплю через Chrome

**Comment by DimonPro242** [comment]
Я приобрел лицензию на Chrome.
Я бы хотел вернуть деньги за лицензию Firefox( так как не пользуюсь этим браузером)

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by DimonPro242** [reply]
Спасибо
